packages/theme/components/FormComponents/EmailInput.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div>
+    <SfInput
+      @click="e => $emit('click', e)"
+      @input="e => $emit('input', e)"
+      v-bind="{...$attrs, ...$props}"
```

Comment:
Is there any reason we have this v-bind property here?

I'd suggest removing the `div` parent and using `SfInput` as the component root. This seems to be more practical and intuitive from the user's perspective.

By doing so, we'll let the user define his own properties and enable full customization based on the component we use for the particular input. Additionally, we should probably remove `@click` and `@input` props if they can be defined directly while using the `EmailInput` component.

Reply:
The root element of a component does not inherit it's props or events if the root element is a vue component ([see here](https://vuejs.org/guide/components/attrs.html#nested-component-inheritance)).

That's why in this case it's necessary to explicitly expose the events and bind the props of `SfInput` component.

The `div` should be removed though.